nestkernel/event.h

```diff
@@ -810,7 +814,19 @@ class CoeffArrayIterator
 class GapJEvent : public SecondaryEvent
 {
 private:
+  /*
+  Conceptual there is an one-to-one mapping between a SecondaryEvent
+  and a SecondaryConnectorModel. The synindex of this particular
+  SecondaryConnectorModel is stored in the static synid_ member variable
+  on model registration. There are however reasons (e.g. the usage of
+  CopyModel or the creation of the labeled synapse model duplicates for pyNN)
+  which make it necessary to register several SecondaryConnectorModels with
+  one SecondaryEvent. Therefore the synindices of all these models are
+  stored  in the synid_vector_. The has_syn_id()-function allows testing
+  if a particular synid is mapped with the SecondaryEvent in question.
+  */
   static synindex synid_;
+  static std::vector< synindex > synid_vector_;
```

Comment:
Usually there is a one-to-one mapping between the secondary connector model and the SecondaryEvent type. The only reasonable exceptions are the duplication for the labeled version of the synapse and each call to `CopyModel`. So I would guess that the size of this vector should never exceed 5 or 6 elements - especially as `CopyModel` might be ineffective for large simulations (keyword: HomConnector/HetConnector).

Would do we conclude on this issue?